app/services/idv/steps/ssn_step.rb

```diff
@@ -41,7 +39,9 @@ def invalid_state_response
 
       def generate_threatmetrix_session_id
         return unless IdentityConfig.store.proofing_device_profiling_collecting_enabled
-        SecureRandom.uuid
+        if flow_session[:threatmetrix_session_id].nil? && !updating_ssn
```

Comment:
I have been thinking about this. I think we want a new session ID each time we render the page, so this can just be:

```suggestion
        if !updating_ssn
```